src/main/java/frc/robot/subsystems/turret/commands/VisionTurret.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.subsystems.turret.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.turret.Turret;
+import frc.robot.utilities.VisionModule;
+
+public class VisionTurret extends CommandBase {
+
+    private Turret turret;
+    private VisionModule visionModule;
+
+    public VisionTurret(Turret turret, VisionModule visionModule) {
+        this.turret = turret;
+        addRequirements(turret);
+        this.visionModule = visionModule;
+    }
+
+    public boolean targetSeen() {
```

Comment:
Document methods

Reply:
I think you should remove this method, you just call another method, there is no purpose for this method instead of calling `visionModule.targetSeen()`